### PR TITLE
[skip ci] (DOCUMENT-951) + (DOCUMENT-950) Update Puppet Server Metrics doc 

### PR DESCRIPTION
**Pull request by clairecadman**
This commit updates the Puppet Server Metrics doc based on DOCUMENT-951 and DOCUMENT-950.

### CODE REVIEW

**Comment by clairecadman** [comment]
@justinstoller I'm not sure if the note on line 29 is what was intended (DOCUMENT-951), if not please let me know how to adjust. Thanks!

File: documentation/puppet_server_metrics.markdown [hunk 1]
```diff
@@ -82,6 +81,8 @@ Grafana runs as a web dashboard, and the `grafanadash` module configures it to u
 
 This loads a dashboard with nine graphs that display various metrics exported from the Puppet Server to the Graphite server. (For details, see [Using the Grafana dashboard](#using-the-sample-grafana-dashboard).) However, these graphs will remain empty until you enable Puppet Server's Graphite metrics.
 
+> Note: If you want to integrate Puppet Server's Grafana exporting with your own infrastructure, use the `grafanadash` module. If you want permanent visualization of metrics, use the `puppetlabs-puppet_metrics_dashboard` module. See [Puppet Metrics Collection](https://github.com/puppetlabs/best-practices/blob/master/puppet-enterprise-metrics-collection.md) for more information.
```
Comment (justinstoller) [inline documentation/puppet_server_metrics.markdown]: I wouldn't use the word "permanent" in "If you want permanent visualization of metrics"

**Comment by justinstoller** [comment]
Line 29 doesn't address DOCUMENT-951, imo.

In more detail:
Puppet Server has the ability to directly export its metrics to a Graphite installation. "grafanadash" is meant to easily set up a graphite instance, configure Puppet Server for exporting to it and visualize the output with grafana. Its meant to help get folks started with Puppet Server's graphite exporting so they can eventually integrate it with their graphite installation. It is also used by the server developers to test our graphite exporting feature, and historically was the easiest way to visualize Puppet Server's metrics.

The "puppet-metrics-dashboard" does not go through our graphite exporting feature. It queries the metrics http api directly and saves the results to disk so that those artifacts can be sent to support for help debugging performance problems. It also includes a docker image of graphite/grafana for easy visualization. For most of its life it was a PE only offering from customer success.

The puppet-metrics-dashboard has been opensourced and has a maintenance commitment from customer success. It represents a lower bar to saving and visualizing Puppet Server metrics and should now be our recommendation to FOSS users in the "I want to easily and save and visualize some Puppet Server metrics".

The grafanadash module will still be used by Puppet Server developers to test our graphite exporting feature for as long as the feature exists, and it *may* still help users prepare a Puppet Server installation for exporting to their existing graphite installation.

imo, DOCUMENT-951 is to make the above distinction and recommendation easily digestible to those who find either the grafanadash or puppet-metrics-dashboard info.

**Comment by justinstoller** [comment]
Also, in the future if you could add `[skip ci]` to the commit title we can skip travis for doc changes.

**Comment by clairecadman** [comment]
Thanks @justinstoller! I added a paragraph based on your description, does this help?

**Pull request merged**